projects/admin/src/app/routes/items-route.ts

```diff
@@ -150,4 +167,40 @@ export class ItemsRoute extends BaseRoute implements RouteInterface {
     }
     return field;
   }
+
+  /**
+   * Populate item record from holding data
+   * @param record: the item record to populate
+   * @param holdingPid: the holding pid
+   */
+  private _populateItemFieldFromHolding(record: any, holdingPid: string) {
+    record.type = 'issue';
+    record.issue = record.issue || {};
+    record.issue.regular = true;  // default to true
+
+    // setting irregular issue from url parameter
+    try {
+      const isIrregular = Boolean(JSON.parse(this._routeToolService.getRouteQueryParam('irregular', 'false')));
```

Comment:
Seems weird boolean(JSON(....

Reply:
This is the best solution I found to validate '1', 'true', 'True', '0', 'false, ... as a boolean value.